InternalError("Throwable(Object(java/lang/InternalError)
  backtrace=java/lang/StackTraceElement[7]
  detailMessage=String(\"Internal error: No constructor found: Test$A.<init>(int)V\")
  cause=Object(class java/lang/InternalError)
  stackTrace=java/lang/StackTraceElement[0]
  depth=int(7)
  suppressedExceptions=Object(class java/util/Collections$EmptyList)
):
stdout: 
stderr: ")